internal/report/report.go

```diff
@@ -270,3 +272,17 @@ const (
 	ExitCodeLow      ExitCode = 101
 	ExitCodeInfo     ExitCode = 100
 )
+
+// collectSummaryMetrics collects the metrics corresponding to the summary.
+func collectSummaryMetrics(sum summary) {
+	metrics.Collect("excluded", sum.excluded)
+	if sum.count == nil {
+		metrics.Collect("vulnerabilities", nil)
+		return
+	}
+	countMetrics := make(map[string]int)
+	for key, value := range sum.count {
+		countMetrics[key.String()] = value
+	}
+	metrics.Collect("vulnerabilities", countMetrics)
```

Comment:
What about something like this? I don't like to repeat twice the string "vulnerabilities". It looks error prone to me.

```suggestion
	var countMetrics map[string]int
	if sum.count != nil {
		countMetrics = make(map[string]int)
		for key, value := range sum.count {
			countMetrics[key.String()] = value
		}
	}
	metrics.Collect("vulnerabilities", countMetrics)
```